specs/deneb/beacon-chain.md

```diff
@@ -89,6 +94,12 @@ and are limited by `MAX_BLOB_GAS_PER_BLOCK // GAS_PER_BLOB`. However the CL limi
 
 ## Configuration
 
+### Validator cycle
+
+| Name | Value |
+| - | - |
+| `MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT` | `uint64(2**3)` (= 8) |
```

Comment:
just a nit: since this new param is only applied from Deneb, should it be called `DENEB_MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT` ?

Reply:
Hmm, but it would break the current code conventions for the new constants/presets/configurations. 🤔

Reply:
🤔 ... It actually should be `MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT_DENEB`, similar to 
`MAX_REQUEST_BLOCKS_DENEB` even if the phase0 counterpart `MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT` won't exist.

Reply:
I see your point, but we do have `MAX_REQUEST_BLOCKS` in phase0! Adding a suffix to a new preset breaks the current code conventions.